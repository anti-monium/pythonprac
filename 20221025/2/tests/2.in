def fib():
	cur, next = 1, 1
	while True:
		yield cur
		cur, next = next, cur + next

i = slide(fib(), 7)
s = next(i)
while s <= 59:
	print(s, end=' ')
	s = next(i)
print()
