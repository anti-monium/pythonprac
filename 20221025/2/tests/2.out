1 1 2 3 5 8 13 1 2 3 5 8 13 21 2 3 5 8 13 21 34 3 5 8 13 21 34 55 5 8 13 21 34 55 
